airflow/dags/sync_ntd_data_xlsx/scrape_ntd_xlsx_urls.py

```diff
@@ -30,20 +38,54 @@ def href_matcher(href):
 
 
 def scrape_ntd_xlsx_urls(**context):
-    for key, value in xlsx_urls.items():
-        url = value
-        req = requests.get(url)
-        soup = BeautifulSoup(req.text, "html.parser")
+    for key, url in xlsx_urls.items():
+        try:
```

Comment:
Since there are others exception handling on line 44 and 55, could those line be unnested and this other "try" moved to line 60?